INSTALL.md

```diff
@@ -105,6 +105,81 @@ For Example:
 ./tools/tpm2_getrandom 4
 ```
 
+### Build example
```

Comment:
I would separate the *installing build dependencies* on a certain distro from the more general *build from source* that should be basically the same for every distro. There's already a *Typical Distro Dependency Installation* section where it explains how to install the dependencies for Ubuntu, so I would just add another sub-section for Fedora.